src/components/Calculator.js

```diff
@@ -4,40 +4,55 @@ import GrayBtn from './GrayBtn';
 import OrangeBtn from './OrangeBtn';
 import BigGrayBtn from './BigGrayBtn';
 import Screen from './Screen';
+import calculate from '../logic/calculate';
 
 class Calculator extends React.Component {
+  constructor() {
+    super();
+    this.state = { };
+    this.btnHandler = this.btnHandler.bind(this);
+  }
+
+  btnHandler(e, keyPressed) {
+    e.stopPropagation();
+    const current = this.state;
+    this.setState(calculate(current, keyPressed));
+    console.log(this.state);
+  }
```

Comment:
- [OPTIONAL] You want to consider removing console.log from your project to follow the best practices in coding. ♻️